site/docs/develop/index.md

```diff
@@ -20,17 +20,17 @@ things such as partitioning (including hidden partitions), schema migrations, ap
 require a pointer to the active metadata set to function. This pointer allows the Iceberg client to acquire and read the
 current schema, files and partitions in the dataset. Iceberg currently relies on the Hive metastore or hdfs to perform
 this role. The requirements for this root pointer store is it must hold (at least) information about the location of the
-current up-to-date metadata file and it must be able to update this location atomically. In Hive this is accomplished by
+current up-to-date metadata file, and it must be able to update this location atomically. In Hive this is accomplished by
 locks and in hdfs by using atomic file swap operations. These operations don’t exist in eventually consistent cloud
 object stores, necessitating a Hive metastore for cloud data lakes. The Nessie system is designed to store the
 root metadata pointer and perform atomic updates to this pointer, obviating the need for a Hive metastore. Removing the
 need for a Hive metastore simplifies deployment and broadens the reach of tools that can work with Iceberg tables.
 The above is specific to how Iceberg behaves however Delta Lake operates in a near identical way.
 
-The Nessie service is a lightweight Java based REST API server. It uses a standard optimistic locking strategy
+The Nessie service is a lightweight, Java based REST API server. It uses a standard optimistic locking strategy
```

Comment:
I'm not sure I agree with this comma, but what do I know :)

I'd change to `lightweight Java-based REST API server`... perhaps the grammar checker thinks the second part is another sentence not an adjective-like construct.

Reply:
My IDE actually said that there should be a comma - but it's also happy with the dash :)